src/components/Hero/HeroSection.jsx

```diff
@@ -0,0 +1,112 @@
+import React from "react";
+import CountUp from "react-countup";
+import GridWrapper from "./GridWrapper";
+import HeroSwiper from "./HeroSwiper";
+
+const HeroSection = () => {
+  return (
+<>
+  {/* Text of the HeroSection */}
+  <div className="flex max-md:grid">
+    <div className="p-10 w-[50%] max-md:w-[100%]">
+      <button className="flex bg-[#F2F4F7] p-[.5rem] rounded-[19px]">
+        <div className="px-3">Visit Our Guidelines</div>
+        {/* Svg for arrow in button */}
+        <svg
+          width="24"
+          height="24"
+          viewBox="0 0 24 24"
+          fill="none"
+          xmlns="http://www.w3.org/2000/svg"
+        >
+          <rect width="24" height="24" rx="12" fill="#E4E7EC"></rect>
+          <path
+            d="M6.16663 12H17.8333M17.8333 12L12 6.16669M17.8333 12L12 17.8334"
+            stroke="#101828"
+            stroke-width="1.5"
+            stroke-linecap="round"
+            stroke-linejoin="round"></path>
+        </svg>
+      </button>
+      <div className="bold font-title text-[2.4rem] max-lg:text-[2rem]">
```

Comment:
this bold doesn't work here. remove it. not needed

Reply:
done